src/test/java/org/jabref/logic/l10n/LocalizationParser.java

```diff
@@ -75,6 +75,14 @@
         }
     }
 
+    public static Set<LocalizationEntry> findLocalizationParametersStringsInJavaFiles(LocalizationBundleForTest type)
+            throws IOException {
+        return Files.walk(Paths.get("src/main"))
```

Comment:
Thanks for the hints, but this is old code which I didnt write and don't wanna change if not really necessary.